doc/source/configuration/shared-file-systems/drivers/vastdata_driver.rst

```diff
@@ -72,10 +75,66 @@ sections that are not discussed in this document:
    vast_root_export = {root_export}
 
 
+Restart of ``manila-share`` service is needed for the configuration
+changes to take effect.
+
+
+Pre-configurations for share support
+--------------------------------------------------
+
+To create a file share you need to:
+
+Create the share type:
+
+    .. code-block:: console
+
+        openstack share type create ${share_type_name} False \
+            --extra-specs share_backend_name=${share_backend_name}
+
+Create NFS share:
+
+    .. code-block:: console
+
+        openstack share create NFS ${size} --name ${share_name} --share-type ${share_type_name}
+
+Pre-Configurations for Snapshot support
+--------------------------------------------------
+
+The following extra specifications need to be configured with share type.
+
+- snapshot_support = True
+
+For new share type, these extra specifications can be set directly when creating share type:
+
+    .. code-block:: console
+
+        openstack share type create ${share_type_name} false \
+            --snapshot-support=true \
+            --extra-specs share_backend_name=${share_backend_name}
+
+Or you can update already existing share type with command:
+
+    .. code-block:: console
+
+        openstack share type set ${share_type_name} --extra-specs snapshot_support=True
+
+
+To snapshot a share and create share from the snapshot
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You need create a share from share type
```

Comment:
The whole paragraph:
Create a share using the share type that has snapshot_support=True specified. Then snapshot the share with the command: